Client/Frontend/Browser/BrowserTrayAnimators.swift

```diff
@@ -3,6 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import UIKit
+import Shared
```

Comment:
Found that GeometryExtensions was in the wrong target so once I moved it we now require pulling in Shared (which makes sense since that file is part of Utils)